packages/jest-runner/src/runTest.ts

```diff
@@ -79,7 +79,10 @@ async function runTestInternal(
   path: Config.Path,
   globalConfig: Config.GlobalConfig,
   config: Config.ProjectConfig,
+
   resolver: ResolverType,
+  sendMessageToJest?: TestFileEvent,
+
   context?: TestRunnerContext,
```

Comment:
```suggestion
  context?: TestRunnerContext,
  sendMessageToJest?: TestFileEvent,
```

to keep arguments in the same order